app/factories/common.py

```diff
@@ -7,8 +7,8 @@
 integer_ = 25
 url_ = 'http://example.com'
 imageUrl_ = 'https://www.w3schools.com/html/pic_mountain.jpg'
-date_ = '2016-12-13T23:59:59.123456+00:00'
-dateEnd_ = '2020-12-14T23:59:59.123456+00:00'
+date_ = '2099-12-13T23:59:59.123456+00:00'
```

Comment:
If you put future date in date start, discount code is going to fail as it won't be active yet

Reply:
@iamareebjamal which date should i put here exactly?